### PR TITLE
[Blog] Comparing Policy as Code Frameworks

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@deepeshaburse this post and @tinniaru3005's don't build properly. Is this something that you're able to fix?

**Comment by tinniaru3005** [comment]
> @deepeshaburse this post and @tinniaru3005's don't build properly. Is this something that you're able to fix?

Yes @leecalcote I am struggling to fix it since one week. @deepeshaburse let me know if there is something you can figure out to fix it.

File: src/collections/blog/2022/12-23/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+is_authorized(user) {
+  user.role in authorization_roles
+}
+
+authorized_to_access_resource(user, resource) {
+  is_authorized(user) && user.has_permission(resource)
+}
+```
+
```
Comment (leecalcote) [inline src/collections/blog/2022/12-23/index.md:104]:
```suggestion

```
function isAuthorized(user: User) -> bool {
  return user.role == "admin" || user.role == "moderator";
}

expression authorizedToAccessResource(user: User, resource: Resource) -> bool {
  return isAuthorized(user) && user.hasPermission(resource);
}
```

In Sentinel, we might define the policy as follows:

```
import "strings"

authorization_roles = ["admin", "moderator"]

is_authorized(user) {
  user.role in authorization_roles
}

authorized_to_access_resource(user, resource) {
  is_authorized(user) && user.has_permission(resource)
}
```

In Rego, we might define the policy as follows:

```
package policy

import data.users

authorization_roles = ["admin", "moderator"]

is_authorized(user) {
  user.role in authorization_roles
}

authorized_to_access_resource(user, resource) {
  is_authorized(user) && user.has_permission(resource)
}
```

```

**Comment by deepeshaburse** [comment]
@leecalcote I checked the error and there's some issue with the integrations index.mdx file of confidential containers. I can't find any error there, any idea why it must be breaking the build?

**Comment by leecalcote** [comment]
> @leecalcote I checked the error and there's some issue with the integrations index.mdx file of confidential containers. I can't find any error there, any idea why it must be breaking the build?

Not to my knowledge given that the current deploy previews are building just fine. Perhaps, this PR's branch needs sync'ed with master.

**Comment by leecalcote** [comment]
> > @leecalcote I checked the error and there's some issue with the integrations index.mdx file of confidential containers. I can't find any error there, any idea why it must be breaking the build?
> 
> Not to my knowledge given that the current deploy previews are building just fine. Perhaps, this PR's branch needs sync'ed with master.

Oh, reading my earlier comment, yes, of course this isn't building - - this is an incomplete blog post that I was looking for someone else to tie off.

**Comment by l5io** [comment]
🚀 Preview for commit 65141959f2a677a83ded35a882620babe29915e8 at: https://63bb1c77014e8249d5f21f35--layer5.netlify.app

**Review by Shivam-AfA — COMMENTED** [review]
@leecalcote This blog seems complete to me and the deploy preview is working fine. Is there something I'm missing?

**Pull request closed without merging**